Documentation/subcommands/stop.md

```diff
@@ -0,0 +1,11 @@
+# rkt stop
```

Comment:
It's scattered through the docs:

* https://github.com/coreos/rkt/blob/6e7e3aba93fda4da0853f29ad7167d1da8f7e016/Documentation/trying-out-rkt.md#launching-an-aci
* https://github.com/coreos/rkt/blob/6e7e3aba93fda4da0853f29ad7167d1da8f7e016/Documentation/running-lkvm-stage1.md#getting-started
* https://github.com/coreos/rkt/blob/8dae5a00f5360f84e50fcb25225603bbba1061f9/Documentation/using-rkt-with-systemd.md#systemd-run
* https://github.com/coreos/rkt/blob/8dae5a00f5360f84e50fcb25225603bbba1061f9/Documentation/using-rkt-with-systemd.md#simple-unit-file

Should we summarize it here?